BackpressureRegulatorStressTest FAILURES

```
java.lang.AssertionError: No error should have been thrown, but StressThread-1 completed error expected null, but was:<com.hazelcast.core.HazelcastOverloadException: Failed to get a callId for invocation: Invocation{ serviceName='null', op=com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest$DummyOperation{serviceName='null', partitionId=2, callId=0, invocationTime=1444514742108, waitTimeout=-1, callTimeout=60000}, partitionId=2, replicaIndex=0, tryCount=250, tryPauseMillis=500, invokeCount=1, callTimeout=60000, target=Address[127.0.0.1]:5702, backupsExpected=0, backupsCompleted=0, connection=Connection [/127.0.0.1:5701 -> /127.0.0.1:51353], endpoint=Address[127.0.0.1]:5702, alive=true, type=MEMBER}>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotNull(Assert.java:755)
    at org.junit.Assert.assertNull(Assert.java:737)
    at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
    at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.test(BackpressureRegulatorStressTest.java:174)
    at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.asyncInvocation(BackpressureRegulatorStressTest.java:74)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-nightly/737/
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-nightly/736/

**BackpressureRegulatorStressTest.asyncInvocation_and_asyncBackups**
```
java.lang.AssertionError: No error should have been thrown, but StressThread-1 completed error expected null, but was:<java.lang.OutOfMemoryError: Java heap space>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.test(BackpressureRegulatorStressTest.java:173)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.asyncInvocation_and_asyncBackups(BackpressureRegulatorStressTest.java:111)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast/1109/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/BackpressureRegulatorStressTest/asyncInvocation_and_asyncBackups/

**BackpressureRegulatorStressTest.asyncInvocation_and_asyncBackups**
```
java.lang.AssertionError: No error should have been thrown, but StressThread-4 completed error expected null, but was:<java.lang.OutOfMemoryError: GC overhead limit exceeded>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.test(BackpressureRegulatorStressTest.java:173)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.asyncInvocation_and_asyncBackups(BackpressureRegulatorStressTest.java:111)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast/1128/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/BackpressureRegulatorStressTest/asyncInvocation_and_asyncBackups/

```
java.lang.AssertionError: No error should have been thrown, but StressThread-5 completed error expected null, but was:<java.lang.OutOfMemoryError: GC overhead limit exceeded>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.test(BackpressureRegulatorStressTest.java:173)
	at com.hazelcast.spi.impl.operationservice.impl.BackpressureRegulatorStressTest.asyncInvocation_and_asyncBackups(BackpressureRegulatorStressTest.java:111)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast/1167/testReport/junit/com.hazelcast.spi.impl.operationservice.impl/BackpressureRegulatorStressTest/asyncInvocation_and_asyncBackups/

--- a/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulatorStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulatorStressTest.java
@@ -28,6 +28,7 @@ import static com.hazelcast.spi.properties.GroupProperty.BACKPRESSURE_ENABLED;
 import static com.hazelcast.spi.properties.GroupProperty.BACKPRESSURE_MAX_CONCURRENT_INVOCATIONS_PER_PARTITION;
 import static com.hazelcast.spi.properties.GroupProperty.BACKPRESSURE_SYNCWINDOW;
 import static com.hazelcast.spi.properties.GroupProperty.OPERATION_BACKUP_TIMEOUT_MILLIS;
+import static com.hazelcast.spi.properties.GroupProperty.PARTITION_COUNT;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static org.junit.Assert.assertEquals;
 
@@ -61,7 +62,9 @@ public class BackpressureRegulatorStressTest extends HazelcastTestSupport {
                 .setProperty(OPERATION_BACKUP_TIMEOUT_MILLIS.getName(), "60000")
                 .setProperty(BACKPRESSURE_ENABLED.getName(), "true")
                 .setProperty(BACKPRESSURE_SYNCWINDOW.getName(), "10")
-                .setProperty(BACKPRESSURE_MAX_CONCURRENT_INVOCATIONS_PER_PARTITION.getName(), "2");
+                .setProperty(BACKPRESSURE_MAX_CONCURRENT_INVOCATIONS_PER_PARTITION.getName(), "2")
+                .setProperty(PARTITION_COUNT.getName(), "10");
+
         HazelcastInstance[] cluster = createHazelcastInstanceFactory(2).newInstances(config);
         local = cluster[0];
         remote = cluster[1];

